variables.tf

```diff
@@ -20,6 +20,11 @@ variable "retention_days" {}
 
 variable "instance_id" {}
 
+variable "block_device_mappings" {
```

Comment:
Add a `description` field

Reply:
(also, please update `README.md`)

Reply:
description and example added as requested, thanks.